app/database/operator/server_data_operator/comparators/index.ts

```diff
@@ -46,7 +46,7 @@ export const isRecordTermsOfServiceEqualToRaw = (record: TermsOfServiceModel, ra
 };
 
 export const isRecordDraftEqualToRaw = (record: DraftModel, raw: Draft) => {
-    return raw.channel_id === record.channelId;
+    return raw.channel_id === record.channelId && raw.root_id === record.channelId;
```

Comment:
I don't get this, should it be:
```suggestion
    return raw.channel_id === record.channelId && raw.root_id === record.rootId;
```

Reply:
You are right. A typo :sweat_smile: